service/history/task/timer_active_task_executor_test.go

```diff
@@ -530,7 +530,7 @@ func (s *timerActiveTaskExecutorSuite) TestProcessActivityTimeout_RetryPolicy_Re
 	s.mockExecutionMgr.On("GetWorkflowExecution", mock.Anything, mock.Anything).Return(&persistence.GetWorkflowExecutionResponse{State: persistenceMutableState}, nil)
 	s.mockExecutionMgr.On("UpdateWorkflowExecution", mock.Anything, mock.Anything).Return(&persistence.UpdateWorkflowExecutionResponse{MutableStateUpdateSessionStats: &persistence.MutableStateUpdateSessionStats{}}, nil).Once()
 
-	s.timeSource.Update(s.now.Add(2 * timerTimeout))
```

Comment:
well that's an unpleasant bug waiting to happen - `now` isn't updated, so calling this 2x wouldn't act like it looked.

🎉 clockwork 🎉